invisible_captcha.gemspec

```diff
@@ -22,4 +22,3 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'test-unit', '~> 3.0'
   spec.add_development_dependency 'byebug'
 end
-
```

Comment:
same ☝️

Reply:
This had 2 new lines, I removed one of them